File: Python/Ex31.py
print("Os vinte primeiros valores da serie de Bergamaschi sao:\n")
n=1
n1=1
n2=1
print(f"{n:.0f}\n"f"{n1:.0f}\n"f"{n2:.0f}")
for i in range (3, 20, 1):
    r=n+n1+n2
    print(f"{r:.0f}")
    n=n1
    n1=n2
    n2=r

File: C/Ex31.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    int a, b, f, n;
    
    a=1;
    b=1;
    f=1;
     getch ();
     printf("Os vinte primeiros valores da serie de Bergamaschi sao: ");
     printf("%d\n%d\n%d\n", a,b,f);
     for (int i=0; i<18; i++){
         n=a+b+f;
         a=b;
         b=f;
         f=n;
         printf("%i\n", n);
         }

         return 0;

         }